docs/videos/EventBusVideo/eventBus.vtt

```diff
@@ -0,0 +1,110 @@
+WEBVTT
+
+STYLE
+::cue {
+  font-family: 'proxima-nova', sans-serif;
+}
+
+1
+00:00:03.500 --> 00:00:9.200

```

Comment:
We may be asked to remove the Ad Hoc branding from the slides - let's see if Anthony says anything about that when we get his review comments.

Reply:
Agreed on removing the branding. It might also be good to have two poster images with consistent titling for each video. When you hit the page there's a lot of perceived white space around the second video. A poster would help that.

Reply:
I was waffling on this as well, but didn't get that as specific feedback, and its not a terrible idea to leave a little mark about who was working on this I think! I have poster images but I'll fix the demo one to be consistent.

Reply:
I've updated the titlecards to be the same format, although I'm not sure how to doublecheck its working since all the previews im doing arent working with the relative files.

Reply:
fwiw -- I've seen technical presentations from other companies at the VA with their branding on it before.